index.html

```diff
@@ -1318,19 +1318,17 @@ <h4 id="h-duration-change-algorithm" resource="#h-duration-change-algorithm"><sp
           <p>Follow these steps when <code><a href="#widl-MediaSource-duration">duration</a></code> needs to change to a <var>new duration</var>.</p>
           <ol>
             <li>If the current value of <code><a href="#widl-MediaSource-duration">duration</a></code> is equal to <var>new duration</var>, then return.</li>
-            <li>Set <var>old duration</var> to the current value of <code><a href="#widl-MediaSource-duration">duration</a></code>.
-            </li><li>Update <code><a href="#widl-MediaSource-duration">duration</a></code> to <var>new duration</var>.</li>
-            <li>If the <var>new duration</var> is less than <var>old duration</var>, then run the
-              <a href="#sourcebuffer-range-removal">range removal</a> algorithm with <var>new duration</var> and
-              <var>old duration</var> as the start and end of the removal range.
-
-              <div class="note"><div class="note-title marker" aria-level="5" role="heading" id="h-note15"><span>Note</span></div><p class="">This preserves audio frames and text cues that start before and end after the <code><a href="#widl-MediaSource-duration">duration</a></code>.</p></div></li>
-            <li>If a user agent is unable to partially render audio frames or text cues that start before and end after the <code><a href="#widl-MediaSource-duration">duration</a></code>, then run the following steps:
+			<li>If <var>new duration</var> is less than the highest starting <a href="#presentation-timestamp">presentation timestamp</a> reported by the <code><a href="#widl-SourceBuffer-buffered">buffered</a></code> attribute across all <a href="#idl-def-SourceBuffer" class="idlType"><code>SourceBuffer</code></a> 
```

Comment:
Please see https://github.com/w3c/media-source/pull/65#discussion_r61780279
Using "highest starting PTS reported by SB.buffered" as the allowed duration reduction lower bound is too lenient: it allows implicit removal of frames in that same buffered range with PTS > the range's "start" time. If that's not what you meant, this is really confusing.
I had thought we agreed that the "HBPT" would be the minimum allowable duration on attempted duration reduction, where HBPT==Highest buffered presentation time across all track buffers, across all SourceBuffers for this MediaSource. HBPT is not necessarily exposed to apps, and it is very rarely going to be the same as any timestamp reported by SourceBuffer.buffered.start(some range index), which is what this current text proposes to use (unless I misread).
Please change to:
If new duration is less than the highest presentation timestamp of any buffered coded frame across all track buffers and SourceBuffer objects for this MediaSource, then throw an ....